frontend/src/admin/lessons/LessonItem.tsx

```diff
@@ -39,13 +43,24 @@ export default function LessonItem(props: {
           >
             <div className='word-group'>
               <h1 className='italic'>word</h1>
-              <h1 className='text-2xl text-purple-400 font-bold'>Valiant</h1>
+              <h1 className='text-2xl text-purple-400 font-bold'>
+                {props.lesson.word}
+              </h1>
             </div>
             <div className='word-group md:mt-0 xs:mt-4'>
               <h1 className='italic'>answer</h1>
-              <h1 className='text-2xl text-purple-400 font-semibold'>
-                Courageous
-              </h1>
+              {props.lesson.choices.map((choice:Choice) => {
```

Comment:
This part is not so readable (i did not test this code, so please reanalyze)

1. Can you create method or variable for this, while making use of `find`, not `map`
```
const correctAnswer = props.lesson.choices.find(choice => choice.is_correct)
```

2. Then you can refer it during render like:
```
<div>{correctAnswer.name}</div>
```